keps/sig-scheduling/5007-device-attach-before-pod-scheduled/kep.yaml

```diff
@@ -0,0 +1,46 @@
+title: DRA Device Attach Before Pod Scheduled
+kep-number: 5007
+authors:
+  - "@KobayashiD27"
+owning-sig: sig-scheduling
+#participating-sigs:
+#  - sig-aaa
+#  - sig-bbb
+status: implementable
+#|implemented|deferred|rejected|withdrawn|replaced
+creation-date: 2024-12-20
+reviewers:
+  - "@pohly"
+approvers:
+  - TBD
+
+see-also:
+  - "/keps/sig-node/4381-dra-structured-parameters"
+  - https://github.com/kubernetes/kubernetes/issues/124042#issuecomment-2548068135
+
+# The target maturity stage in the current dev cycle for this KEP.
+stage: alpha
+#|beta|stable
+
+# The most recent milestone for which work toward delivery of this KEP has been
+# done. This can be the current (upcoming) milestone, if it is being actively
+# worked on.
+latest-milestone: "v1.33"
+
+# The milestone at which this feature was, or is targeted to be, at each stage.
+milestone:
+  alpha: "v1.33"
+  beta: "v1.34"
+  stable: "v1.35"
+
+# The following PRR answers are required at alpha release
+# List the feature gate name and the components for which it must be enabled
+feature-gates:
+  - name: DRADeviceAttachBeforePodScheduled
```

Comment:
Perhaps a bit shorter?

"DRADeviceAttachDuringScheduling"

Reply:
Thank you, I use your suggestion.